frontend/src/metabase/query_builder/components/view/sidebars/ChartTypeSidebar.tsx

```diff
@@ -24,7 +25,7 @@ import {
   SettingsButton,
 } from "./ChartTypeSidebar.styled";
 
-const DEFAULT_ORDER = [
+const DEFAULT_ORDER: CardDisplayType[] = [
```

Comment:
note: as this is already defined `as const` we can derive the type from this. We can leave it as-is though.

`type CardDisplayType = typeof ORDERED_CARD_DISPLAY_TYPES[number]`